packages/builders/src/components/selectMenu/UnsafeSelectMenu.ts

```diff
@@ -11,7 +11,7 @@ export class UnsafeSelectMenuBuilder extends ComponentBuilder<
 	/**
 	 * The options within this select menu
 	 */
-	protected readonly options: UnsafeSelectMenuOptionBuilder[];
+	public readonly options: UnsafeSelectMenuOptionBuilder[];
```

Comment:
can't comment on other files but there's something similar in [`UnsafeModalBuilder`](https://github.com/discordjs/discord.js/blob/main/packages/builders/src/interactions/modals/UnsafeModal.ts#L9) where you can change `protected readonly data` to `public readonly data`. Also data is being [cloned twice](https://github.com/discordjs/discord.js/blob/main/packages/builders/src/interactions/modals/UnsafeModal.ts#L12-L13) in the other file, the constructor can simply do `this.data = data;`